src/SFML/Graphics/Font.cpp

```diff
@@ -129,7 +129,7 @@ m_fontHandles(copy.m_fontHandles),
 m_isSmooth(copy.m_isSmooth),
 m_info(copy.m_info),
 m_pages(copy.m_pages),
-m_pixelBuffer(copy.m_pixelBuffer)
+m_pixelBuffer(copy.m_pixelBuffer.begin(), copy.m_pixelBuffer.end())
```

Comment:
How does this relate to the nullptr checks you added?

Reply:
Dang cherry-pick...